src/isconnected.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
This is a one liner (basically). I think this can just be in the subprocess call in Python. I don't think we need another file for it.